hicommonwealth/commonwealth
# Updated admin sidebar links and added new pages scaffhold

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6347

## Description of Changes
- Updated links in the community sidebar for admins
- Added scaffold for new admin pages

This PR doesn't aim to update sidebar CSS.

<img width="245" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/5ac429ec-aa25-4c36-94ec-d3a3f4ca172c">


## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_NEW_ADMIN_ONBOARDING` in `.env`
- Visit any community you are an admin off.
- Verify you see the new links in admin sidebar
- Verify that clicking on each link redirects to that page.

## Deployment Plan
N/A

## Other Considerations
N/A
